lib/hanami/configuration.rb

```diff
@@ -56,7 +56,7 @@ def initialize(application_name:, env:)
 
       @actions = load_dependent_config("hanami/action") {
         require_relative "configuration/actions"
-        Actions.new
+        Actions.new(assets_server_url: assets.server_url)
```

Comment:
There's a couple of problems with this line:

1. If hanami-asets isn't available, then `assets` will be a `NullConfiguration` instance, and will raise a `NoMethodError` for `#server_url`.
2. By referencing the server_url directly, if the user ends up ever changing this, we will end up using a stale value of the setting in the actions config.

We probably want to pass the whole `assets` object into `Actions.new` directly, and then refer to the `assets.server_url` in the `Actions#finalize!` method.

@katafrakt — would you mind having a go at that?

Reply:
@timriley I changed it to pass whole assets configuration, but it gets ugly, because `NullConfiguration` might be not loaded at this point (I assume this it by design?). Also not sure what you meant by referencing in `finalize!` - should the whole CSP-related logic move there?